ideainspect.groovy

```diff
@@ -100,6 +107,9 @@ println "#"
 println "Executing: " + ideaArgs.join(" ")
 
 def processBuilder = new ProcessBuilder(ideaArgs)
+if(cliOpts.sc){
+  processBuilder.environment().put("idea.analyze.scope", cliOpts.sc)
+}
```

Comment:
Out of curiosity: We do need to pass the property both, on the process environment _and_ the `idea.properties` file? Or is this line rather for fail-safety?